src/Microsoft.Tye.Hosting/Model/JSON/ReplicaStatusJson.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Tye.Hosting.Model
+{
```

Comment:
I'm not a fan of just appending JSON to the end of these names. Anything better?

Reply:
I'd use a separate namespace and possibly prepend V1 to all of the things that are serialization DTOs.

Reply:
Alright, I'll fully commit to it.